src/librustdoc/passes/stripper.rs

```diff
@@ -93,8 +93,8 @@ impl<'a> DocFolder for Stripper<'a> {
 
             // implementations of traits are always public.
             clean::ImplItem(ref imp) if imp.trait_.is_some() => true,
-            // Struct variant fields have inherited visibility
-            clean::VariantItem(clean::Variant::Struct(..)) => true,
+            // Variant fields have inherited visibility
+            clean::VariantItem(clean::Variant::Struct(..) | clean::Variant::Tuple(..)) => true,
```

Comment:
This seems like an unrelated—but still good—fix, right?

Reply:
Ah indeed, didn't even pay attention. But yes, bug fix! :)

Reply:
Or actually it shouldn't have any effect on behavior, but it *is* a performance fix :)